domains/fathulfahmy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "fathulfahmy",
+        "email": "mfathulfahmy@gmail.com"
+    },
+    "record": {
+        "URL": "fathulfahmy.netlify.app"
```

Comment:
Invalid URL record (missing protocol)

Reply:
Thank you for notifying me, I have updated the url with `https://` protocol